pkg/networkmanager/wireguard.jsx

```diff
@@ -315,7 +325,21 @@ export function WireGuardDialog({ settings, connection, dev }) {
                                     id={idPrefix + '-endpoint-peer-' + i}
                                 />
                             </FormGroup>
-                            <FormGroup className='pf-m-3-col-on-md' label={_("Allowed IPs")} fieldId={idPrefix + '-allowedips-peer-' + i}>
+                            <FormGroup className='pf-m-3-col-on-md' label={_("Keepalive")} fieldId={idPrefix + '-keepalive-peer-' + i}>
+                                <InputGroup>
+                                    <InputGroupItem isFill>
+                                        <TextInput
+                                            value={peer.persistentKeepalive}
+                                            onChange={(_, val) => {
+                                                setPeers(peers => peers.map((peer, index) => i === index ? { ...peer, persistentKeepalive: val } : peer));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19521-20231024-110617-ce68b93e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)